src/init.js

```diff
@@ -0,0 +1,43 @@
+// Command to set up Cavy within a repo
+// `cavy init`
+//
+const indexContent = require('../templates/index.test.js');
+const specContent = require('../templates/exampleSpec.js');
+const { existsSync, writeFileSync, mkdirSync } = require('fs');
+
+DEFAULT_TEST_DIR = 'specs';
+DEFAULT_ENTRY_FILE = 'index.test.js';
+
+// Sets up Cavy entry index file and example spec.
+function init(args) {
+  console.log('cavy: Adding Cavy to your project...');
+
+  // The first additional argument to cavy init is the name of the test
+  // directory. This is used in the index.test.js file too.
+  const folderName = args[1] || DEFAULT_TEST_DIR;
+
+  if (existsSync(folderName)) {
+    console.log(`cavy: Looks like a ./${folderName} directory already exists for this project.`);
+    console.log('cavy: To continue set up, re-run the command with an alternative test directory name: `cavy init <test-directory>`')
+    process.exit(0)
+  }
+
+  // Create a exampleTest.js file in the specified folder in the route of the
+  // project.
+  mkdirSync(`./${folderName}`);
```

Comment:
1. Can we sanitise this input?
2. Do we need the `./`? Does that break this on Windows?

```
» node cavy.js init foo/bar
cavy: Adding Cavy to your project...
fs.js:121
    throw err;
    ^

Error: ENOENT: no such file or directory, mkdir './foo/bar'
```

Reply:
As per below, perhaps disregard my point about `./`.

Reply:
Agreed, should test on Windows, a couple of people have raised issues that I think could be down to Windows environments. And if the GitHub dev survey is anything to go by...

Reply:
Oh now interesting, maybe I shouldn't have used the word sanitise. Because now if I did `cavy init myproject/specs` will I end up with a folder called `myprojectspecs`? I think it'd be better to return an error to avoid anything unexpected.

Reply:
Agreed, @jalada - now exits if folder name invalid.